static/js/pages/committee-single.js

```diff
@@ -12,6 +12,55 @@ var helpers = require('../modules/helpers');
 
 var singlePageTableDOM = 't<"results-info results-info--bottom meta-box"frip>';
 
+var tableOpts = {
+  dom: singlePageTableDOM,
+  pagingType: 'simple',
+  lengthChange: false,
+  pageLength: 10,
+  useHideNull: false
+};
+
+var sizeInfo = {
+  0: {limits: [0, 199.99], label: 'Under $200'},
+  200: {limits: [200, 499.99], label: '$200 - $499'},
+  500: {limits: [500, 999.99], label: '$500 - $999'},
+  1000: {limits: [1000, 1999.99], label: '$1000 - $1999'},
+  2000: {limits: [2000, null], label: 'Over $2000'},
+};
+var sizeColumns = [
+  {
+    data: 'size',
+    width: '50%',
+    className: 'all',
+    render: function(data, type, row, meta) {
+      var info = sizeInfo[data];
+      var anchor = document.createElement('a');
+      anchor.textContent = info.label;
+      var uri = URI('/receipts')
+        .query({
+          committee_id: row.committee_id,
+          min_amount: info.limits[0],
+          max_amount: info.limits[1]
+        });
+      anchor.setAttribute('href', tables.buildAggregateUrl(uri, row.cycle));
```

Comment:
I didn't specify this before, but this should be on the `total` column, not the `size` column, to keep it consistent with the others tables.